frontend/src/p5/Gridshot.js

```diff
@@ -145,10 +148,15 @@ export const Gridshot = (p, gamemodeDataFilePath, dispatch) => {
                 if (p.mouseX > 0 && p.mouseX < 800 && p.mouseY > 0 && p.mouseY < 600){
                     // Initialize timer
                     timer = 3;
-                    dispatch({ type: 'SET_TIMER', payload: timer });
+                    timerId = setInterval(handleTimer, 1000);
+                    dispatch({  
+                        type: 'SET_TIMER', 
+                        payload: {timer: timer, timerId: timerId}
+                    });

```

Comment:
In the future, just format multi-line one line statements like this, I think its more readable this way (the way you did it is obviously still very readable, however this new way might allow us to be a little bit more consistent in our code style)

```
dispatch({  
    type: 'SET_TIMER', 
    payload: {
        timer: timer, 
        timerId: timerId
    }
});
```